canonical-websites/www.ubuntu.com
# Updated the BootStack service description

## Done

- Updated the BootStack service description
- Started creating includes so that some sections can be reused in the Ubuntu Advantage SD updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [new BootStack sd](http://0.0.0.0:8001/legal/bootstack/service-description) and compare to the [copy doc](https://docs.google.com/document/d/1wiUeISVo9jJ2hVPRiXKnnX__R3eyFfJ5PB15-v9bEAM/edit#heading=h.c18ukqn9x9da)
    - please note, this copy might vary *slightly* based on new includes that are part of a larger restructure
- also look at [the archived version](http://0.0.0.0:8001/legal/bootstack/2015-07-01-service-description) and see that it has a link back to the latest version

## Issue / Card

Fixes #2715
